### ISSUE TITLE
Automatic Backup is not working (cron not started by supervisor)

### ISSUE THREAD

**Issue by martmaiste** [nordic-institute/X-Road-Security-Server-sidecar#87]
Automatic Backup feature is not working unless the cron daemon is manually started inside docker container or cron section is manually added to supervisor config.

Temporary fix was to create a copy of the `/etc/supervisor/conf.d/xroad.conf`, add the section for cron and mount the file as a volume.

```
[program:cron]
command = cron -f -L 15
autostart=true
autorestart=true

```
The problem is present on at least the following docker images:
niis/xroad-security-server-sidecar:7.1.1
niis/xroad-security-server:focal-7.1.0

**Comment by raits** [comment]
Thank you for reporting this issue to us. I have added this as a bug to our backlog so that the development team can look into the issue. We aim to fix it by the next release.

The JIRA item can be found here: https://nordic-institute.atlassian.net/browse/XRDDEV-2161

**Comment by raits** [comment]
Happy to report that the issue has been resolved by the team and the fix will be included in version 7.2.0 of the X-Road Security Server Sidecar.
